src/Math/include/BipedalLocomotion/Math/Skew.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file Skew.h
+ * @authors Giulio Romualdi
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#ifndef BIPEDAL_LOCOMOTION_MATH_SKEW_H
+#define BIPEDAL_LOCOMOTION_MATH_SKEW_H
+
+#include <type_traits>
+
+#include <Eigen/Dense>
+
+namespace BipedalLocomotion
+{
+namespace Math
+{
+
+/**
+ * Return a 2x2 skew matrix given a scalar.
+ * @param v scalar value
+ * @return a 2x2 skew matrix
+ * @note The skew symmetric matrix is given by:
+ * \f[
+ * v^\hat = \begin{bmatrix}
+ * 0 & -v \\
+ * v & 0
+ * \end{bmatrix}
+ * \f]
+ */
+template <typename _Scalar>
+typename std::enable_if<std::is_arithmetic<_Scalar>::value, Eigen::Matrix<_Scalar, 2, 2>>::type
+skew(const _Scalar v)
+{
+    return (Eigen::Matrix<_Scalar, 2, 2>() << _Scalar(0.), -v, v, _Scalar(0.)).finished();
+}
+
+/**
+ * Return a 3x3 skew matrix given a scalar.
```

Comment:
It is not a scalar.

Reply:
Fixed here: https://github.com/dic-iit/bipedal-locomotion-framework/pull/352/commits/9a5c03370b1bdb9288711b7a0b8fca871571c3f8